pkg/converter/tool/nydusd.go

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) 2022. Nydus Developers. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
Should this live in `pkg/nydussdk` instead?

Reply:
Only use it in converter tests, moved to `tests/nydusd.go`.